database/download.go

```diff
@@ -34,7 +34,7 @@ type DownloadResponseDTO struct {
 	Skylink   string    `bson:"skylink" json:"skylink"`
 	Name      string    `bson:"name" json:"name"`
 	Size      uint64    `bson:"size" json:"size"`
-	Timestamp time.Time `bson:"timestamp" json:"downloadedOn"`
+	Timestamp time.Time `bson:"created_at" json:"downloadedOn"`
```

Comment:
```suggestion
	CreatedAt time.Time `bson:"created_at" json:"downloadedOn"`
```

Reply:
The `json` doesn't match, either

Reply:
Yes, we changed course after this was implemented. I'll rename the field in the IDE, rather than merging this, so it changes all the uses.